File: 프로그래머스/1/12906. 같은 숫자는 싫어/같은 숫자는 싫어.py
def solution(arr):

    answer = []
    #빈배열
    for i in range(len(arr)):
        # arr의 길이만큼 반복
        answer.append(arr[i])
        # answer에 arr[i]번째를 추가
        if [arr[i]] == arr[i + 1: i + 2]:
            #만약 arr[i]번째가 arr의 i + 1번 중에 있다면
            answer.pop()
            #answer에서 제거
    return answer

File: 프로그래머스/1/12906. 같은 숫자는 싫어/같은 숫자는 싫어.js
function solution(arr)
{
    var answer = [];

    console.log(arr.length)
    for(let i = 0; i < arr.length; i++){
        
       answer.push(arr[i])
        if(arr[i] == arr[i + 1]){
           answer.pop()
        }
        
    }

    
    return answer;
}